src/components/banner/banner.styles.scss

```diff
@@ -8,13 +8,13 @@
 
   .banner-content {
     max-width: 960px;
-    display: flex;  
+    display: flex;
 
     .banner-eevee {
-      width: 141px;
-      height: 160px;
       background: url('./images/eevee.png') no-repeat 0 0;
       background-size: cover;
+      height: 160px;
+      width: 141px;
```

Comment:
nit: abc

Reply:
Made a keyboard shortcut to alphabetize. Done